src/sentry/static/sentry/app/utils/withPlugins.tsx

```diff
@@ -17,14 +17,14 @@ type WithPluginProps = {
 };
 
 type InjectedPluginProps = {
-  plugins: Plugin[];
+  plugins: {plugins: Plugin[]; loading: boolean};
```

Comment:
Ah was this typed incorrectly?

Reply:
@billyvg yea, the original is wrong. See:
https://github.com/getsentry/sentry/blob/27ed4c63910fdf84481cde13269a0e33e4f3ff5b/src/sentry/static/sentry/app/views/settings/projectPlugins/details.jsx#L144-L147